docs/docusaurus/docs/reference/learn/data_quality_use_cases/unstructured_data.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: 'Unstructured data'
+title: 'Validate unstructured data with GX Cloud'
+description: 'Learn how to validate metadata from scanned PDFs using GX Cloud'
```

Comment:
descriptions should end in periods. Apologies for not indicating this when I shared the syntax for descriptions 

```suggestion
description: 'Learn how to validate metadata from scanned PDFs using GX Cloud.'
```